datadog/resource_datadog_security_monitoring_default_rule.go

```diff
@@ -256,9 +280,45 @@ func buildSecMonDefaultRuleUpdatePayload(currentState datadogV2.SecurityMonitori
 	}
 	payload.Filters = payloadFilters
 
+	tfOptions := d.Get("options").([]map[string]interface{})
+	payloadOptions := datadogV2.NewSecurityMonitoringRuleOptions()
+	emptyOptions := true
+
+	if v, ok := tfOptions[0]["decrease_criticality_based_on_env"]; ok {
+		payloadOptions.SetDecreaseCriticalityBasedOnEnv(v.(bool))
+		emptyOptions = false
+	}
+
+	if !emptyOptions {
+		payload.Options = payloadOptions
+	}
```

Comment:
I would reuse the pattern from [here](https://github.com/DataDog/terraform-provider-datadog/blob/master/datadog/resource_datadog_security_monitoring_rule.go#L369) so it's easier to update and add options in the future

Reply:
Good point, thanks 👌